core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -723,6 +732,7 @@ public SchemaRegistryConfig(ConfigDef configDef, Properties props) throws RestCo
     if (compatibilityType == null) {
       throw new RestConfigException("Unknown compatibility level: " + compatibilityTypeString);
     }
+    buildMetricsContextLabels();
```

Comment:
why move the logic here?

Reply:
there are two classes of importance that are using the metrics context KafkaSchemaRegistry and RestApplication (from cc-utils). The issue currently is SR builds the context labels when SR is starting but RestApplication uses the config metrics context on construction (which is why the rest app metrics like jersey and jetty were not showing up). SR uses the metrics container which sets defaults to what is in the config.